component/acme-dns.libsonnet

```
@@ -99,8 +98,13 @@ local mountpaths = {
   acmednsjson: '/etc/acme-dns',
   scripts: '/scripts',
 };
+local fqdnStripWildcard(fqdns) = [
+  // Strips the first 2 characters `*.` from the string if it starts with `*.`
+  if std.startsWith(fqdn, '*.') then std.slice(fqdn, 2, std.length(fqdn), 1) else fqdn
```

Probably clearer with `std.substr()`:

```suggestion
  if std.startsWith(fqdn, '*.') then std.substr(fqdn, 2, std.length(fqdn)) else fqdn
```

---

I was searching for that, but somehow it slipped 🤷‍♂️